docker/healthcheck.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+database_dir=/usr/share/GeoIP
+if ! [ -z "$GEOIPUPDATE_DB_DIR" ]; then
+  database_dir=$GEOIPUPDATE_DB_DIR
+fi
+
+# The health check is done by checking if the database directory is modified within the
+# update period minus 1 minute. The 1 minute is a threshold for allowing slower starts.
+# Without the LockFile in the database directory, this check is not going to be working
+# since database files are not going to be modified when there are no updates.
+cutoff_date=$(($(date +%s) - $GEOIPUPDATE_FREQUENCY * 60 * 61 ))
```

Comment:
`$GEOIPUPDATE_FREQUENCY * 60 * 61` ends up adjusting by more than the 1 minute stated on line 9. The actual value ends up being `$GEOIPUPDATE_FREQUENCY` minutes.